scripts/install.js

```diff
@@ -37,7 +37,17 @@ function shouldRenderProgressBar() {
 
 function getDownloadUrl(platform, arch) {
   const releasesUrl = `${CDN_URL}/${pkgInfo.version}/sentry-cli`;
-  const archString = arch.indexOf('64') > -1 ? 'x86_64' : 'i686';
+  let archString = '';
+  switch (arch) {
+    case 'x64':
+      archString = 'x86_64';
+      break;
+    case 'x86':
```

Comment:
i install @sentry/cli failed with node@12.18.3 32bit

because sentry only provide sentry-cli-Windows-i686.exe 

but by your rule, install script will try to download file sentry-cli-Windows-ia32.exe ...

Reply:
@zyf0330 @sqxieshuai it's a one-line fix which I'd greatly appreciate :)

Reply:
I guess adding `case 'ia32':` after should do the trick?

Reply:
You are right, use 'ia32' as replacement of 'x86'.Could you issue a PR?

Reply:
Done in #824